students/njschafi/Lesson03/mailroom.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+## NEIMA SCHAFI, LESSON 3 Assignment - mailroom part1
+
+##Using lists as the data structure since they are mutable (can change the contents).
+donors = [['Leon Dechino',200.43,30.23,1.50],['Rupert Everton',23.99],['Zordon Lagasse',400.00, 500.00, 600.00],['Kavinsky Mega', 23.23,100.45],['Corey Camelot', 1.00, 2.00]]
+donorNames = [x[0] for x in donors] ## puts all names into a sequence
+def menu():
+    """Main menu function which user can select one of the options shown"""
+    selection = input('\nPlease select one of the following actions (select number):\n 1. Send a Thank You\n 2. Create a Report\n 3. Quit\n Your selection: ')

```

Comment:
this entry point should be in a `while True` loop, and then you won't have to call `menu` in each function.